src/state/selectors.ts

```diff
@@ -0,0 +1,46 @@
+// Utility selectors, which are only used for params selecting and do NOT select from the store
+
+import type { AccountId, AssetId, ChainId } from '@shapeshiftoss/caip'
+import createCachedSelector from 're-reselect'
+
+import type { ReduxState } from './reducer'
+import type { PubKey } from './slices/validatorDataSlice/validatorDataSlice'
+
+// List of all the params filter consumed with selectParamFromFilter
+type ParamFilter = {
+  accountAddress: string
+  contractAddress: string
+  assetId: AssetId
+  accountId: AccountId
+  accountNumber: number
+  chainId: ChainId
+  validatorAddress: PubKey
+}
+
+// List of all the params filter consumed with selectParamFromFilterOptional
```

Comment:
Eventually could be made generic/granular when we end up consuming these more, just consolidates the portfolio and foxEth slices params into one for now